src/app/screens/Send/index.tsx

```diff
@@ -126,6 +126,7 @@ function Send() {
           <div className="pt-4">
             <TextField
               id="invoice"
+              type="text"
```

Comment:
Why did you add the type everywhere? It's "text" by default?

Reply:
we have type added as text at few of the text field and few of them doesn't contain them. so shall we remove type field at all the fields containing type="text"

it's better to be consistent!

Reply:
having type explicitly in the code would be better. isn't it?